edison/static/js/policy.controller.js

```diff
@@ -5,6 +5,61 @@ import { PolicyElementsModifier } from './policyElementsModifier.js'
 var policy = new Policy();
 var countCondition = 0;
 
+var policiesExamples = [];
```

Comment:
I'm not sure it's a good idea to set here the example policy. if i understand correctly, this is only used for the Demo, so maybe put all the parts related to this policiesExample in a different module or class, so you can handle it from a single place- it's more maintainable and decrease the amount of bugs

Reply:
Ok, done :)